### PR TITLE
✨update state fields

### PR DESCRIPTION
**Pull request by Jephuff**

### CODE REVIEW

File: modules/server/src/utils/getIndexPrefix.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+function getIndexPrefix({ projectId, type }) {
```
Comment (alex-wilmer) [inline modules/server/src/utils/getIndexPrefix.js]: why take the object type here?

Comment (Jephuff) [reply modules/server/src/utils/getIndexPrefix.js]: instead of just index?

Comment (Jephuff) [reply modules/server/src/utils/getIndexPrefix.js]: I updated to just pass index 👍  initially I had thought I would have type available everywhere, but that was wrong. this makes more sense! :)

Comment (alex-wilmer) [reply modules/server/src/utils/getIndexPrefix.js]: yea it looks like you're only using `index` here, and in some places above passing `type : { index }`. was just wondering if maybe we would use it later

**Pull request merged**